Assets/LeapMotion/Core/Scripts/VR/LeapVRTemporalWarping.cs

```diff
@@ -69,127 +38,180 @@ protected struct TransformData {
+    }
 
     [Tooltip("Allows smooth enabling or disabling of the Rotational warping of Leap Space.  Usually should match image warping.")]
     [Range(0, 1)]
     [SerializeField]
-    private float tweenRotationalWarping = 0f;
+    private float _tweenRotationalWarping = 0f;
+    public float tweenRotationalWarping {
+      get {
+        return _tweenRotationalWarping;
+      }
+      set {
+        _tweenRotationalWarping = Mathf.Clamp01(value);
+      }
+    }
 
     [Tooltip("Allows smooth enabling or disabling of the Positional warping of Leap Space.  Usually should be disabled when using image warping.")]
     [Range(0, 1)]
     [SerializeField]
-    private float tweenPositionalWarping = 0f;
+    private float _tweenPositionalWarping = 0f;
+    public float tweenPositionalWarping {
+      get {
+        return _tweenPositionalWarping;
+      }
+      set {
+        _tweenPositionalWarping = Mathf.Clamp01(value);
+      }
+    }
 
     [Tooltip("Controls when this script synchronizes the time warp of images.  Use LowLatency for VR, and SyncWithImages for AR.")]
     [SerializeField]
-    private SyncMode syncMode = SyncMode.LOW_LATENCY;
+    private SyncMode _syncMode = SyncMode.LOW_LATENCY;
+    public SyncMode temporalSyncMode {
+      get {
+        return _syncMode;
+      }
+      set {
+        _syncMode = value;
+      }
+    }
 
     [Header("Advanced")]
     [Tooltip("Forces an update of the temporal warping to happen in Late Update using the existing head transform.  Should not be enabled if you " +
              "are using the default Unity VR integration!")]
     [SerializeField]
-    private bool forceCustomUpdate = false;
+    private bool _forceCustomUpdate = false;
 
     // Manual Time Alignment
     [Tooltip("Allow manual adjustment of the rewind time.")]
     [SerializeField]
-    private bool allowManualTimeAlignment;
+    private bool _allowManualTimeAlignment;
 
     [Tooltip("Timestamps and other uncertanties can lead to sub-optimal alignment, this value can be tuned to get desired alignment.")]
     [SerializeField]
-    private int warpingAdjustment = 60; //Milliseconds
+    private int _warpingAdjustment = 60; //Milliseconds
+    public float warpingAdjustment {
+      get {
+        return _warpingAdjustment;
+      }
+    }
 
     [SerializeField]
-    private KeyCode unlockHold = KeyCode.RightShift;
+    private KeyCode _unlockHold = KeyCode.RightShift;
 
     [SerializeField]
-    private KeyCode moreRewind = KeyCode.LeftArrow;
+    private KeyCode _moreRewind = KeyCode.LeftArrow;
 
     [SerializeField]
-    private KeyCode lessRewind = KeyCode.RightArrow;
+    private KeyCode _lessRewind = KeyCode.RightArrow;
 
-    public float TweenImageWarping {
-      get {
-        return tweenImageWarping;
-      }
-      set {
-        tweenImageWarping = Mathf.Clamp01(value);
-      }
-    }
+    // Manual Device Offset
+    [Tooltip("Allow manual adjustment of the Leap device's virtual offset and tilt. These "
```

Comment:
Hmm wanted to discuss a workflow question.  Are we ok with this boolean just showing/hiding the fields, not actually changing the way things work?  For example if someone somehow gets something into a bad state, they might not know that they need to _check_ the toggle in order to fix it.